### ISSUE TITLE
User-friendly way of adding problems to a practical

### ISSUE THREAD

**Issue by antonwsu** [tomitrescak/clara-support#45]
In preparation for other organisations joining, it would be good to develop a user-friendly way of adding problems to a practical. What users essentially want is seeing the list of unique problems for a given topic (e.g. loops) and adding those quickly. I've started numbering them like Problem 3_1 to make this task a bit simpler, but it's still not straight-forward. Try creating a new practical on loops and populate it with problems that haven't been in use this semester. It will take quite some time and for new users it will be quite a challenge. How can we make this process easier?

**Comment by tomitrescak** [comment]
Anton, it's already there ;) You just have not notices it.
You can add multiple tags to your exercise in the exercise admin page:

![image](https://user-images.githubusercontent.com/2682705/110423087-911e7500-80f4-11eb-9769-ca7c656ac883.png)

Then, in the practical admin page you can search by Tag:

![image](https://user-images.githubusercontent.com/2682705/110423182-b90dd880-80f4-11eb-9e67-a1cab430834f.png)

It even tells you where that given exercise is used in case you are reusing it:

![image](https://user-images.githubusercontent.com/2682705/110423252-d5aa1080-80f4-11eb-9ed3-2cfd080c1e35.png)

**Comment by tomitrescak** [comment]
Let me know if this is sufficient.

**Comment by antonwsu** [comment]
This might be sufficient. I didn't know about the tag. Good idea. I saw the window that shows where the exercise was used, but it's quite messy. Is there a way to only display unique problems (don't show duplicates). We also need a good way to preview these exercises (maybe show a picture or something)? The process was still very time-consuming. I had to add exercise, go inside, remove, repeat multiple times before I compiled a practical following this approach. I haven't used the tags, but I searched for something like Problem 3_1 and it only gave me problems from practical 3. There were many duplicates and no obvious way to see what the problem is without adding it, saving going inside and then deleting. Having a picture would certainly help.
